com.mastermindzh.tidal-hifi.yml

```diff
@@ -0,0 +1,62 @@
+---
+app-id: com.mastermindzh.tidal-hifi
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+build-options:
+  cflags: '-O2 -g'
+  cxxflags: '-O2 -g'
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+  append-path: /usr/lib/sdk/node16/bin
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=pulseaudio'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--filesystem=xdg-run/app/com.discordapp.Discord:create'
+  - '--own-name=org.mpris.MediaPlayer2.tidal-hifi'
+  - '--talk-name=org.freedesktop.Notifications'
+modules:
+  - name: tidal-hifi
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/Mastermindzh/tidal-hifi
+        tag: 2.8.0
+        commit: 89589b75e1cdbecbeffbbe7933696bc330106d35
+        dest: main
+      - generated-sources.json
+      - type: file
+        path: run.sh
+      - type: file
+        path: com.mastermindzh.tidal-hifi.desktop
+      - type: file
+        path: com.mastermindzh.tidal-hifi.appdata.xml
+      - type: file
+        path: icon.png
+        dest-filename: com.mastermindzh.tidal-hifi.png
```

Comment:
you not using the icon in the upstream repository?

Reply:
The upstream icon is a non-standard size